tests/Client/UpdaterTest.php

```diff
@@ -979,28 +953,14 @@ public function providerExceptionForInvalidMetadata(): array
                     'targets' => 2,
                 ],
             ],
-            [
-                // § 5.3.4
-                '4.root.json',
```

Comment:
This test case doesn't apply to the inconsistent fixture, since it doesn't have a `4.root.json`, so it only appears in `ConsistentFixturesUpdaterTest`.